### PR TITLE
Feature/log map enrich reports

### PR DESCRIPTION
**Pull request by moltude**
Adds proof of concept reporting to enrichment chain. Only providing "detail" reporting for the language field.

### CODE REVIEW

File: src/main/scala/dpla/ingestion3/executors/EnrichExecutor.scala [hunk 1]
```diff
@@ -92,33 +102,65 @@ trait EnrichExecutor extends Serializable {
...
+    logger.info(EnrichmentSummary.getSummary(summaryData))
+
     sc.stop()
 
     // Clean up checkpoint directory, created above
     Utils.deleteRecursively(new File("/tmp/checkpoint"))
 
     // Log error messages.
-    failures.foreach(msg => logger.warn(s"Enrichment error >> $msg"))
+    failures.foreach(logger.error(_))
```
Comment (AudreyAltman) [inline src/main/scala/dpla/ingestion3/executors/EnrichExecutor.scala]: What does line 140 do?  Does it write to a log file?

File: src/main/scala/dpla/ingestion3/executors/MappingExecutor.scala [hunk 2]
```diff
@@ -76,91 +79,120 @@ trait MappingExecutor extends Serializable {
...
-
-    logger.info(MappingSummary.getSummary(mappingSummary))
-    logger.info(s"Number of exceptions ${failures.length}")
-
-    // Write warn and error messages to CSV files. These should all share the same timestamp, minor work TBD
     val baseLogDir = s"$dataOut/../logs/"
-    val time = timeInMs.toString
 
-    val exceptions = sc.parallelize(failures).toDS()
+    val exceptionsDS = sc.parallelize(exceptions).toDS()
```
Comment (AudreyAltman) [inline src/main/scala/dpla/ingestion3/executors/MappingExecutor.scala]: Do you need to make this into a `DataSet`?  Would it be easier to work with it as a plain `RDD`?  Not a big deal, just curious.

File: src/main/scala/dpla/ingestion3/reports/summary/ReportFormattingUtils.scala [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package dpla.ingestion3.reports.summary
+
+import org.apache.commons.lang.StringUtils
+
+/**
+  *
+  */
```
Comment (AudreyAltman) [inline src/main/scala/dpla/ingestion3/reports/summary/ReportFormattingUtils.scala]: Empty comment block.

File: src/main/scala/dpla/ingestion3/executors/EnrichExecutor.scala (hunk 1 above)
Comment (moltude) [reply src/main/scala/dpla/ingestion3/executors/EnrichExecutor.scala]: Exactly

**Pull request merged**